python/api/src/araali/__main__.py

```diff
@@ -653,6 +700,19 @@ def aws(args):
 
     parser_ctl = top_subparsers.add_parser("ctl", help="run araalictl commands")
 
+    parser_quickstart = top_subparsers.add_parser("quickstart", help="launch quickstart for given email")
+    quickstart_subparsers = parser_quickstart.add_subparsers(dest="quickstart_subparser_name")
+
+    parser_quickstart_vm = quickstart_subparsers.add_parser("vm", help="launch VM quickstart")
+    parser_quickstart_vm.add_argument('-e', '--email', help="email of quickstart user")
+    parser_quickstart_vm.add_argument('-k', '--key', help="ssh key to use")
+
+    parser_quickstart_eks = quickstart_subparsers.add_parser("eks", help="launch EKS quickstart")
+    parser_quickstart_eks.add_argument('-e', '--email', help="email of quickstart user")
+    parser_quickstart_eks.add_argument('-lC', '--launchcluster', action="store_true", help="need to launch a new cluster if set to true")
```

Comment:
two letter `-lC` doesnt work very well. You dont have to have a short form. `-lC `is a shortform for `-l -C`, so it gets confusing. That is why the `--longform` was invented.

The default should be to launch. You can also use eks command to auto derive if its an existing clusters: `aws eks list-clusters` and act accordingly